server/channels/app/platform/helper_test.go

```diff
@@ -32,6 +33,7 @@ type TestHelper struct {
 	// BasicPost    *model.Post
 
 	SystemAdminUser *model.User
+	Tb              testing.TB
```

Comment:
Let's use just `T` as that is the same variable used in the rest of the code base.
```suggestion
	T              testing.TB
```